### ISSUE TITLE
d.ts files are wrong

### ISSUE THREAD

**Issue by pvinis** [relay-tools/relay-hooks#99]
Take `useQuery` for example.

`useQuery.ts` has
```
export const useQuery = <TOperationType extends OperationType = OperationType>(
...
```

After `yarn build`, `useQuery.d.ts` has
```
export declare const useQuery: <TOperationType extends any = any>(gqlQuery: any, variables?: TOperationType["variables"], options?: QueryOptions) => RenderProps<TOperationType>;
```

Notice the `extends` part. In the d.ts file it should still be `OperationType` instead of `any`.

**Comment by morrys** [comment]
thanks for the tip :)
In these days I will investigate the problem

**Comment by morrys** [comment]
hi @pvinis,
I just officially released [version 3.5.2](https://github.com/relay-tools/relay-hooks/releases/tag/v3.5.2) which fixes the d.ts files.

For any other problem, open another issue 👍
